### DISCUSSION TITLE
Missing VAT on invoice

### DISCUSSION THREAD

**Discussion by dandan2611** [community/community#124077] [Enterprise] [answered]
Hello everyone!

I recently set up a GitHub organization for a company I work for. We took the "Teams" offer and used our company credit cards informations.
For accounting and legal reasons, we need to have our VAT number on invoices we receive so we can justify the spent amount.

However, GitHub did not generate an invoice (or receipt) with the VAT number of the company after paying the subscription. I noticed that the VAT information input box was pretty deep under billing informations.

I contacted GitHub via their support website ([support.github.com](https://support.github.com/)) to ask if it was possible to generate another receipt with the correct informations. However, no answer event after 3 weeks.
I decided to ask here if anyone has already been in this kind of situation, or if there is any way to regenerate the correct receipt automatically.

Thanks for your future answers!
Adam.

**Comment by dandan2611** [comment, marked as answer]
When editing a company VAT informations, the previous receipts automatically update themselves. Fixed!

**Reply by MinglrApp** [reply]
where can I find this setting in the organization profile?

**Reply by dandan2611** [reply]
In your "Billing and plans" section from your org!
![image](https://github.com/community/community/assets/19997696/1b491e4a-1933-4a27-8af0-ab263dd563aa)

**Comment by boobonke** [comment]
Hi Adam,
We have a similar issue.
We receive a receipt rather than a VAT invoice.
Have you found a way to receive invoices?
Thanks

**Reply by dandan2611** [reply]
Did not find a way to receive invoices unfortunately instead of receipt.
Seems we need to use invoices via bank from highest offer in order to have a proper one :(

**Comment by synevopl** [comment]
Hi
We have a Team Free account and bought Github Copilot for 2 seats for $38 but the invoice was not generated. Can I ask for it to be generated and information on how to download it?
Thanks in advance for your help :)
